### DISCUSSION TITLE
"Spherical" earth possible in an .earth file?

### DISCUSSION THREAD

**Discussion by miles-tbe** [pelicanmapping/osgearth#1896] [Support Q&A]
I have a need for being able to display a globe in both a WGS84 oblate and spherical context. I currently only know of how to set an .earth file up for wgs84, like so:

```xml
<options>
   <profile>
      <srs>wgs84</srs>
   <profile>
``` 

If there exists a way to change this for a spherical Earth I would like to know. 

Thanks!

**Comment by gwaldron** [comment]
You can set a profile using a PROJ4 string like so:
```xml
        <profile>
            <srs>+proj=longlat +R=6378137 +no_defs</srs>
        </profile>
```

**Reply by miles-tbe** [reply]
I tried this but with no luck. Was this meant to be the exact srs string to try or just an example?

**Reply by gwaldron** [reply]
This is the exact string. Can you elaborate on "no luck"? Are you using osgEarth 3.2?  Here is a complete earth file:
```xml
<map>
    <options>
        <profile>
            <srs>+proj=longlat +R=6378137 +no_defs</srs>
        </profile>
    </options>
    
    <GDALImage name="World GeoTIFF">
        <url>../data/world.tif</url>
    </GDALImage>
</map>
```

**Reply by miles-tbe** [reply]
I am using osgEarth 3.1.0. The issue was not with the globe itself not being loaded in but the objects I have drawn on the globe can either be drawn with locations on a spherical earth or aon an oblate (WGS84) earth and when I do the spherical earth for my object they are still not matching up with where I expected them with that string. But my earth files have been using the version 2 earth file method:

```xml
<map version="2">
    <options>
        <profile>
            <srs> +proj=longlat +R=6378137 +no_defs</srs>
        </profile>
        <terrain>
            <driver>rex</driver>
            <tile_size>7</tile_size>
        </terrain>
    </options>
    <elevation>
        <url>SRTM/EU.tif</url>
        <driver>gdal</driver>
    </elevation>
    <image>
        <url>BlueMarbe/Marble.tif</url>
        <driver>gdal</driver>
    </image>
</map>
```

Would switching to the new method make a difference? Whether that be for this particular question, or in general?
